<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gamelevel</class>
 <widget class="QMainWindow" name="Gamelevel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/image/box.png</normaloff>:/image/box.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{border-radius:100px;border:1px solid gray;background-color:#aaffff}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="levelsmallp">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="choose">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>请选择关卡！</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关卡一</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关卡二</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关卡三</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关卡四</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关卡五</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关卡六</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关卡七</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose8">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关卡八</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose9">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关卡九</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose10">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关卡十</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose11">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关卡十一</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose12">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关卡十二</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>levelsmallp</zorder>
   <zorder>choose</zorder>
   <zorder>choose1</zorder>
   <zorder>choose2</zorder>
   <zorder>choose3</zorder>
   <zorder>choose4</zorder>
   <zorder>choose5</zorder>
   <zorder>choose6</zorder>
   <zorder>choose7</zorder>
   <zorder>choose8</zorder>
   <zorder>choose9</zorder>
   <zorder>choose10</zorder>
   <zorder>choose11</zorder>
   <zorder>choose12</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="start"/>
    <addaction name="settings"/>
    <addaction name="help"/>
    <addaction name="separator"/>
    <addaction name="custom"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="start">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/smallp.png</normaloff>:/image/smallp.png</iconset>
   </property>
   <property name="text">
    <string>开始(R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="settings">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/settings.png</normaloff>:/image/settings.png</iconset>
   </property>
   <property name="text">
    <string>设置(S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="help">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/help.png</normaloff>:/image/help.png</iconset>
   </property>
   <property name="text">
    <string>帮助(H)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="custom">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/custom.jpg</normaloff>:/image/custom.jpg</iconset>
   </property>
   <property name="text">
    <string>自定义关卡</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
